extensions.toml

```
@@ -1416,6 +1416,10 @@ version = "0.0.6"
 submodule = "extensions/muted"
 version = "0.0.1"
 
+[my-extension]
```

We need to use the extension ID here:

```suggestion
[mcp-server-byterover]
```